scripts/add_rtp_task_resource_records.py

```diff
@@ -0,0 +1,166 @@
+#! /usr/bin/env python
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2021 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+"""Add RTP task resource records to the M&C database.
+
+This script finds all entries in RTPTaskJobID (RTPTaskMultipleJobID) that
+have no corresponding entry in RTPTaskResourceRecord
+(RTPTaskMultipleResourceRecord).  Computing information (time, memory, cpu
+usage) for all missing entries is obtained from the SLURM database and added
+to the corresponding M&C task resource record tables.
+
+"""
+
+import numpy as np
+from subprocess import check_output
+from astropy.time import Time
+from hera_mc import mc
+from hera_mc.rtp import (
+    RTPTaskJobID,
+    RTPTaskResourceRecord,
+    RTPTaskMultipleJobID,
+    RTPTaskMultipleResourceRecord,
+)
+
+
+def cpu_time_to_seconds(time_str):
+    """
+    Convert `time_str` from 'D-HH:MM:SS' (format of CPUTime from SLURM) to
+    seconds.
+
+    Parameters
+    ----------
+    time_str : str
+        String with format 'D-HH:MM:SS' or 'HH:MM:SS'.
+
+    Returns
+    -------
+    secs : float
+        Number of seconds.
+
+    """
+    if "-" in time_str:
+        days = float(time_str.split("-")[0])
+        time_str = time_str.split("-")[1]
+    else:
+        days = 0.0
+
+    if len(time_str.split(":")) == 2:
+        mins, secs = time_str.split(":")
+        mins = float(mins)
+        secs = float(secs)
+    else:
+        hours, mins, secs = time_str.split(":")
+        hours = float(hours)
+        mins = float(mins)
+        secs = float(secs)
+        secs += hours * 3600.0
+    secs += mins * 60 + days * 24 * 60 * 60
+    return secs
+
+
+def query_slurm_db(jobid):
+    """
+    Query the SLURM database using `sacct` and the `--format` flag
```

Comment:
This docstring should also fit on a single line. It's okay to have a second block underneath that spans multiple lines, but the first line should be self-contained. (This is consistent with the [numpy docstring format](https://numpydoc.readthedocs.io/en/latest/format.html) that we use.)